engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5761,11 +5767,18 @@ void demon_hunter_t::create_buffs()
                          set_bonuses.t30_havoc_2pc->ok() ? set_bonuses.t30_havoc_2pc_buff : spell_data_t::not_found() )
           ->set_default_value_from_effect_type( A_MOD_TOTAL_STAT_PERCENTAGE )
           ->set_pct_buff_type( STAT_PCT_BUFF_AGILITY );
-  
+
   buff.t30_havoc_4pc =
       make_buff<buff_t>( this, "seething_potential",
                          set_bonuses.t30_havoc_4pc->ok() ? set_bonuses.t30_havoc_4pc_buff : spell_data_t::not_found() )
           ->set_default_value_from_effect( 1 );
+
+  buff.t30_vengeance_2pc = make_buff<buff_t>(
```

Comment:
Given the description "Multiple applications may overlap." is this a normal stacking buff or should this be `->set_stack_behavior( buff_stack_behavior::ASYNCHRONOUS )`? Usually they only use this wording for async behavior.